react-native: add page (#5208)
tldr-pages/tldr: pages/common/react-native.md

# react-native

> A framework for building native apps with React.
> More information: <https://reactnative.dev>.

- Initialize a new React Native project in a directory of the same name:

`react-native init {{project_name}}`

- Start the metro bundler:

`react-native start`

- Start the metro bundler with a clean cache:

`react-native start --reset-cache`

> - Build your application and start it on a connected Android device or emulator:

Reviewer: Generally we try to avoid the 2nd person (e.g. `you`, `your`, etc) in tldr-pages :slightly_smiling_face:
```suggestion
- Build the current application and start it on a connected Android device or emulator:
```

`react-native run-android`

> - Build your application and start it on iOS simulator:

Reviewer 2: ```suggestion
- Build your application and start it on an iOS simulator:
```

> - Build your application and start it on an iOS simulator:

Reviewer: Same here.
```suggestion
- Build the current application and start it on an iOS simulator:
```

`react-native run-ios`

> - Build your application on `release` mode and start it on a connected Android device or emulator:

Reviewer: Same here.
```suggestion
- Build the current application in `release` mode and start it on a connected Android device or emulator:
```

`react-native run-android --variant={{release}}`

- Start `logkitty` and print logs to stdout:

`react-native log-android`

> - Start iOS device syslog tail and print logs to stdout:

Reviewer 2: ```suggestion
- Start syslog tail on an iOS simulator and print logs to stdout:
```

> - Start syslog tail on an iOS simulator and print logs to stdout:

Reviewer: Is `syslog tail` a command that is started on the simulator? If so, it should be enclosed in backticks.

Author: Based on https://docs.expo.io/workflow/logging/
```suggestion
- Start `tail system.log` for an iOS simulator and print logs to stdout:
```
